tests/Resources/ti.media.videoplayer.addontest.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions'),
+	utilities = require('./utilities/utilities');
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.media.videoplayer.addontest.js;
  Line: 11;
-->

- :warning: tests/Resources/ti.media.videoplayer.addontest.js line 11 – 'utilities' is assigned a value but never used. (no-unused-vars)